scss/layout/_print.scss

```diff
@@ -7,12 +7,21 @@
 // Inlined to avoid the additional HTTP request: h5bp.com/r
 // ==========================================================================
 
+//variables
+$print-background-after: transparent !default;
+$print-color-after: #000 !default;
```

Comment:
I would not add the "-after" part here. The styles are applied to all elements including before and after.